src/expr.h

```diff
@@ -78,9 +78,10 @@ class Expr : public ASTNode {
 
     /** If this is a constant expression that can be converted to a
         constant of the given type, this method should return the
-        corresponding llvm::Constant value.  Otherwise it should return
-        NULL. */
-    virtual llvm::Constant *GetConstant(const Type *type) const;
+        corresponding llvm::Constant value and a flag denoting if it's
+        valid for multi-target compilation for use as an initializer.
```

Comment:
for use as an initializer **of a global variable**.

Locals are not a problem.

Reply:
Done